resolwe/flow/views/data.py

```diff
@@ -184,6 +159,29 @@ def get_or_create(self, request, *args, **kwargs):
         kwargs['get_or_create'] = True
         return self.create(request, *args, **kwargs)
 
+    def perform_get_or_create(self, request, *args, **kwargs):
+        """Perform "get_or_create" - return existing object if found."""
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        process = serializer.validated_data.get('process')
+        process_input = request.data.get('input', {})
+
+        # use default values if they are not given
```

Comment:
This can/should be moved to model's utils in the future.

Reply:
In the future or now? :-)

Reply:
You can do it now. [Data model](https://github.com/genialis/resolwe/blob/master/resolwe/flow/models/data.py#L353) uses the same code.